### ISSUE TITLE
proper way to render page

### ISSUE THREAD

**Issue by decarlomw** [10clouds/FluidBottomNavigation-rn#5]
Hi I am trying to implement this within my app, I got the tabs to show but how do I link the pages to the tab. I pretty new to react-native so please forgive me if there is a simple solution.

**Comment by dpatel** [comment]
Did you figure out the solution? I am thinking to use this library.

**Comment by diegoHdzZ** [comment]
Yes I did, i changed some things from the original. Heres the repo  [https://github.com/diego3799/CustomBottomTabNavigator](url)
